backend/workflow/internal_scrape_workflow.py

```
@@ -0,0 +1,338 @@
+This module implements a new LangGraph workflow using internal scrape.py tools:
+1. User query analysis
+2. Question classification (general vs. search-required)
+3. Search execution with search_detailed_products tool
+4. Data compatibility parsing for legacy format
+5. Final response generation
+
+This workflow provides better performance and control over data extraction.
+"""
+
+# Load environment variables at module level to ensure they're available
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)